apps/src/util/whyDidYouRender.js

```diff
@@ -0,0 +1,22 @@
+import experiments from '@cdo/apps/util/experiments';
+import React from 'react';
+
+/**
+ * This experiment loads the why-did-you-render library to help track rendering
+ * performance issues: https://github.com/welldone-software/why-did-you-render
+ *
+ * To enable it for a specific component, add `static whyDidYouRender = true`
+ * at the top of the component class.
+ *
+ * Note: once we upgrade to React 16, we should upgrade this library.
+ */
+if (
+  process.env.NODE_ENV === 'development' &&
+  experiments.isEnabled(experiments.WHY_DID_YOU_RENDER)
+) {
+  const whyDidYouRender = require('@welldone-software/why-did-you-render');
```

Comment:
This is definitely optional but... now that we're talking about it, I guess you could try using `import(...)` syntax to see if it still works :-) from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import:
```
(async () => {
  if (somethingIsTrue) {
    // import module for side effects
    await import('/modules/my-module.js');
  }
})();
```